### ISSUE TITLE
thread 'main' panicked at 'byte index 39 is out of bounds of `AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA`', 

### ISSUE THREAD

**Issue by lskatz** [hcdenbakker/colorid#35]
Error when running `colorid search` with k=39, and the read is less than my kmer length

**Comment by lskatz** [comment]
```Counting k-mers, this may take a while!
411 kmers in query
No perfect hits!
Counting k-mers, this may take a while!
411 kmers in query
No perfect hits!
Counting k-mers, this may take a while!
thread 'main' panicked at 'byte index 39 is out of bounds of `AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA`', libcore/str/mod.rs:2094:9
note: Run with `RUST_BACKTRACE=1` for a backtrace.```

**Comment by hcdenbakker** [comment]
This should be fixed in version 1.4!
